### PR TITLE
package: add nydus to release artifacts

### PR DESCRIPTION
**Pull request by liubin**
Install nydus related binaries under /opt/kata/libexec/

Fixes: #5726

Signed-off-by: Bin Liu <bin@hyper.sh>

### CODE REVIEW

**Review by bergwolf — APPROVED** [review]
lgtm! Thanks!

**Review by fidencio — APPROVED** [review]
lgtm, thanks @liubin!

**Comment by fidencio** [comment]
> @liubin, @bergwolf, do you think we should bother about having a specific config file for nydus instead of virtiofsd?

I'm asking as I think it'd be beneficial for users to simply use a different runtimeClass, such as `kata-nydus`, and be able to simply test it out.

**Comment by liubin** [comment]
@fidencio The nydusd is only for file sharing, we have already supported it by configuring the `configuraiton.toml`:

https://github.com/kata-containers/kata-containers/blob/5cbf879659532cad3df084eb3c89a584ac19d2ba/src/runtime/config/configuration-qemu.toml.in#L176-L180


The `runtimeClass` is more related to snapshotter, that is what [`nydus-snapshotter`](https://github.com/containerd/nydus-snapshotter) is doing, so IMO there no more changes are needed.

In the new containerd (from 1.7), it [supports specifying snapshotter for runtime](https://github.com/containerd/containerd/pull/6899), so we can use different snapshotters for different runtimes.

**Comment by fidencio** [comment]
> @fidencio The nydusd is only for file sharing, we have already supported it by configuring the `configuraiton.toml`:

Let me rephrase what I wanted to say.

I think we should have a configuration using nydus by default, something like configuration-qemu-nydus.toml, and then add a runtimeClass to use such configuration.  So we don't have users having to connect to the node and changing configuration in order to be able to test nydus.
